gslib/tests/test_boto_util.py

```diff
@@ -52,3 +53,58 @@ def testSetsHostBaseToMtlsIfClientCertificateEnabled(self,
         domain='',
         password='password',
     )
+
+  def testHasConfiguredCredentialsNoCreds(self):
```

Comment:
So this one is probably failing on XML because it is probably reading the config from a parent config. You might have to do something like this

https://github.com/GoogleCloudPlatform/gsutil/blob/9b3767664199da0c71ac59f77fd4a9c0fbb43d78/gslib/tests/test_shim_util.py#L42

For now you can just copy over that function and use it. I can roll out a refactor later on to move this function to gslib.tests.util and use it in both these module

You will probably need to use it for all the other tests here as well.

Reply:
I found another solution that directly targets the method being tested